docs/sql-ref-syntax-aux-resource-mgmt-add-jar.md

```diff
@@ -33,15 +33,30 @@ ADD JAR file_name
 
 * **file_name**
 
-    The name of the JAR file to be added. It could be either on a local file system or a distributed file system.
+    The name of the JAR file to be added. It could be either on a local file system or a distributed file system or an ivy URL.
+    Apache Ivy is a popular dependency manager focusing on flexibility and simplicity. Now we support two parameter in URL query  string:
 
+      1. transitive: whether to download dependent jars related to your ivy URL.
+      2. exclude: exclusion list when download ivy URL jar and dependent jars.
```

Comment:
When listing something in the doc, `*` seems to be used in most places, I think. e.g., https://github.com/apache/spark/blame/master/docs/sql-ref-syntax-aux-show-tables.md#L45-L48

cc: @huaxingao

Reply:
Updated